### PR TITLE
FAST: initial implementation of lightweight tenants

### PR DESCRIPTION
**Pull request by ludoo**
This has been lightly tested with Julio, and an initial README section added. IMHO it could be merged as a preliminary implementation.

### CODE REVIEW

**Review by sruffilli — APPROVED** [review]
Great stuff!

File: fast/stages/1-resman/README.md [hunk 1]
```diff
@@ -166,6 +167,93 @@ tags = {
 }
 ```
 
+### Lightweight multitenancy
+
+If the organization needs to support tenants without the full complexity and separation offered by our [full multitenant support](../../stages-multitenant/), this stage offers a simplified setup which is suitable for cases where tenants have less autonomy, and don't need to implement FAST stages inside their reserved partition.
+
+This mode is activated by defining tenants in the `tenants` variable, while IAM configurations that apply to every tenant can be optionally set in the `tenants_config` variable.
```
[the remark this answers is not in the document]
Comment (ludoo) [reply fast/stages/1-resman/README.md:174]: > How tricky is it to add those to IAM.tf?

Please explain, do you mean IAM.md? That is really easy to do.

[the remark this answers is not in the document]
Comment (sruffilli) [reply fast/stages/1-resman/README.md:174]: 🤦 ofc i meant IAM.md

File: fast/stages/1-resman/README.md [hunk 2]
```diff
@@ -166,6 +167,93 @@ tags = {
...
+        - GCS buckets
+        - service account
+        - IAM bindings()
+    }
+```
+
+This is an example that shows how to populate the relevant variables.
+
+```hcl
+tenants = {
```
[the remark this answers is not in the document]
Comment (ludoo) [reply fast/stages/1-resman/README.md:235]:
```
  all_drs_domains = concat(
    [var.organization.customer_id],
    try(local.policy_configs.allowed_policy_member_domains, []),
    compact([for k, v in var.tenants : try(v.organization.customer_id, "")])
  )
  ```
  
  it's already taken care of internally

**Review by juliocc — APPROVED** [review]

**Pull request merged**